ui/src/components/Tasks/validate.js

```diff
@@ -24,15 +24,15 @@ const validate = (values, props) => {
   // here we update the resolution limits based on the energy the typed in the form,
   // the limits come from a table sent by the client
 
-  const validFname = /^[\w#[\]{}-\-]+$/u.test(props.filename);
+  const validFname = /^[-_#{}[\]\w]+$/u.test(props.filename);
 
   const emptyField = 'field is empty';
 
   if (!validFname) {
     errors.prefix = INVALID_CHAR_MSG;
   }
 
-  if (props.subdir && !/^[\w/{}-\-]+$/u.test(props.subdir)) {
+  if (props.subdir && !/^[-{}\/\w]+$/u.test(props.subdir)) {
```

Comment:
Initial regex was `/^[-\w\-\/\_\{\}]+$/`. Same problem with the dash: it has to be at the front and doesn't need to be specified twice.